cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -71,7 +71,9 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
    */
   function init() {
     var triggerClickedBinded = _triggerClicked.bind( this );
+    var triggerOverBinded = _triggerOver.bind( this );
```

Comment:
Removed them in https://github.com/cfpb/cfgov-refresh/commit/7a1bd7ea7db4e9117f8055b6096b38767ee8b8f0 and just set the binding in the call. The variable is only needed where there are repeated calls to the binded function, or if it needs to be removed.